src/main/java/org/broadinstitute/hellbender/utils/read/GATKRead.java

```diff
@@ -486,6 +500,22 @@ default int numCigarElements(){
      */
     void setMateIsUnmapped();
 
+    /**
+     * @return True if this read's mate is unmapped, not including reads where the position is set but the read is marked as unmapped. Otherwise false.
+     */
+    boolean mateIsUnplaced();
```

Comment:
Unfortunately I can't provide a default implementation here because getContig() masks empty values. The whole point of this PR was to provide better access to the contig index and start position in samrecord beyond what getContig(), and getStart() currently provide.

Reply:
you're right that there isn't a getMateAssignedContig.  Should there be?